TEKDB/TEKDB/tekdb_filebrowser.py

```diff
@@ -0,0 +1,269 @@
+import os
```

Comment:
is `TEKDB/TEKDB/tekdb_filebrowser.py` the correct location for this file? I wasn't totally sure.